addons/OPT/events.hpp

```diff
@@ -0,0 +1,15 @@
+/*
+List all events used by OPT here
+*/
+#define OPT_EVENT(var)                  QUOTE(TRIPLE(PREFIX,event,var))
```

Comment:
Man könnte sich überlegen, ob man hier hardcoded `opt` anstatt dem `PREFIX` macro verwendet. Sollte man nämlich mal auf eines der Events außerhalb der Mod zugreifen wollen (z.B. inm init-Feld einer Einheit im Editor), muss man den String ja manuell eintragen und wenn sich `PREFIX` ändern sollte, dann ändern sich natürlich auch gleich alle Event-Namen. Wenn man den prefix dagegen hardcoded rein schreibt, ist dem nicht so :man_shrugging:

Reply:
Das ist schon eine seeeeehr spezielle Ausnahme :D . Aber jo, why not